src/backend/api/__init__.py

```diff
@@ -75,6 +76,13 @@ async def conn(self, attempts: int = 1) -> AsyncGenerator[S3Client, None]:
         `S3Client`'s operations, and convert them to `fastapi.HTTPException`.
         """
         session = AioSession()
+        session.register(
+            "creating-client-class.s3",
+            boto3.utils.lazy_call(
+                "aioboto3.s3.inject.inject_s3_transfer_methods"
+            ),
+        )
```

Comment:
Can we get a brief comment explaining what's happening, so our future-selves won't hate us? :)